src/richie/apps/courses/templates/courses/cms/fragment_course_runs.html

```diff
@@ -0,0 +1,148 @@
+{% load cms_tags i18n extra_tags %}
+
+<div class="course-detail__aside__runs">
+{% if current_page.publisher_is_draft %}
+  <h2 class="course-detail__aside__runs__title">{% trans 'Course runs' %}</h2>
+  {% for run in course.get_course_runs_for_language %}
```

Comment:
Why display the draft differently? It seems confusing.

Reply:
This ensure that you have access to all the course run and their information when in draft. The course runs without date (to be planned) never show up in the published view as it is.
Would you rather have them always seperated like in the published version and add a section for «to be planned» course run ?